README.md

```diff
@@ -49,12 +49,31 @@ You can find the created distribution under **/target/smarthome-packaging-sample
 
 4. Start runtime
 ================
+
+The minimum requirement for running this distribution is:
+
+* JavaSE 8, JavaSE Embedded 8, or Azul Zulu-Embedded 8
+* When running embedded profiles: at least compact 2 profile
```

Comment:
Doesn't this contradict https://github.com/eclipse/smarthome-packaging-sample/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R143?